frontend/apps/app/package.json

```diff
@@ -85,6 +89,7 @@
     "lint:tsc": "tsc --noEmit",
     "postinstall": "cp ../../packages/db-structure/node_modules/@ruby/prism/src/prism.wasm prism.wasm",
     "start": "next start",
+    "sync-vector-store": "tsx scripts/syncVectorStore.ts",
```

Comment:
- Is this intentional? → `scripts/syncVectorStore.ts`
- I’m not quite sure how [this commit](https://github.com/liam-hq/liam/pull/1721/commits/5b79a3f531487f23af3c5cb44d3bf66ba2bfeeb7) is supposed to work! Could you clarify?

Reply:
Oh, thank you for noticing. 
I was left with the code that was originally developed to allow embedding with the command. I will fix it!🙏

Reply:
https://github.com/liam-hq/liam/pull/1721/commits/b3b4509ab1d3003aaebf6127cd6b22e54518d441
I removed the unwanted scripts and the `tsx` package I had put in to run them!